app/services/doc_auth/lexis_nexis/responses/true_id_response.rb

```diff
@@ -257,6 +257,11 @@ def combine_alert_data(all_alerts, alert_name)
             new_alert_data[:name] = value if key.end_with?('_AlertName')
             new_alert_data[:result] = value if key.end_with?('_AuthenticationResult')
             new_alert_data[:region] = value if key.end_with?('_Regions')
+            new_alert_data[:disposition] = value if key.end_with?('_Disposition')
```

Comment:
should we be testing these new keys in `spec/services/doc_auth/lexis_nexis/responses/true_id_response_spec.rb`?

Reply:
I see. If you think it's not worth testing that's okay - as it's at the core of the change I thought it might be worth testing, but maybe that was not the best call. Thank you for adding it, though!

Reply:
@night-jellyfish  i did added tests for those as mentioned.